app/controllers/message_drafts/signature_requests_controller.rb

```diff
@@ -46,11 +46,13 @@ def set_message_draft
     end
 
     def signers_scope
-      Current.tenant.signer_group.users.order(name: :asc)
+      signer_groups_ids = (Current.tenant.signer_group.users.map(&:user_group) + [Current.tenant.signer_group]).map(&:id)
+      Group.where(id: signer_groups_ids).order(name: :asc)
```

Comment:
Tu dajme pre istotu aj scope na tenanta, niekto nahodou zmeni to nad tym a bude to leakovat.